docs/content/en/docs/yaml-crd-ref/taskdefinition.md

```diff
@@ -95,147 +101,42 @@ but timeouts seem to be measured in seconds.
+    [Synopsis for python-runtime runner](#python-runtime)
     * **container** -- Use the runner defined
       for the `container-runtime` container.
       This is a standard Kubernetes container
       for which you define the image, runner, runtime parameters, etc.
       and code the functionality to match the container you define.
       See
-      [Yaml synopsis for container-runtime container](#yaml-synopsis-for-container-runtime).
-  * **retries** (optional) - specifies the number of times,
+      [Synopsis for container-runtime container](#synopsis-for-container-runtime).
+  * **retries** (optional) -- specifies the number of times
     a job executing the `KeptnTaskDefinition`
     should be restarted if an attempt is unsuccessful.
-  * **timeout** (optional)* -- specifies the maximum time, in seconds,
+  * **timeout** (optional) -- specifies the maximum time, in seconds,
     to wait for the task to be completed successfully.
     If the task does not complete successfully within this time frame,
     it is considered to be failed.
 
-## Yaml Synopsis for deno-runtime container
-
-When using the `deno-runtime` runner to define a task,
-the task is coded in Deno-script
-(which is mostly the same as JavaScript and TypeScript)
-and executed in the
-[Deno](https://deno.com/manual) runner,
-which is a lightweight runtime environment
-that executes in your namespace.
-Note that Deno has tighter restrictions
-for permissions and importing data
-so a script that works properly elsewhere
-may not function out of the box when run in the `deno-runtime` runner.
-
-```yaml
-apiVersion: lifecycle.keptn.sh/v?alpha?
-kind: KeptnTaskDefinition
-metadata:
-  name: <task-name>
-spec:
-  deno:
-    inline | httpRef | functionRef | ConfigMapRef
-    parameters:
-      map:
-        textMessage: "This is my configuration"
-    secureParameters:
-      secret: slack-token
-```
-
-### Spec fields for deno-runtime definitions
-
-* **spec**
-  * **deno** -- Specify that the task uses the `deno-runtime`
-    and is expressed as a [Deno](https://deno.com/) script.
-    Refer to [deno runtime](https://github.com/keptn/lifecycle-toolkit/tree/main/runtimes/deno-runtime)
-    for more information about this runner.
+## Synopsis for container-runtime
 
-    The task can be defined as one of the following:
+Use the `container-runtime` to specify your own
+[Kubernetes container](https://kubernetes.io/docs/concepts/containers/)
+and define the task you want to execute.
 
-    * **inline** - Include the actual executable code to execute.
-      This can be written as a full-fledged Deno script
-      that is included in this file.
-      For example:
+You can code a sequence of executables here
+that need to be run in order
+as long as they are executables that are part of the lifecycle workflow.
+Task sequences that are not part of the lifecycle workflow
+should be handled by the pipeline engine tools being used
+such as Jenkins, Argo Workflows, Flux, and Tekton.
```

Comment:
What use-case do you want to describe with this part ?

Reply:
This was a quote from @thisthat in response to someone trying to migrate from Keptn v1 to Keptn

Reply:
ok, thank you